README.md

```diff
@@ -1,5 +1,16 @@
 # AvN Node
 
+## Deprecation Notice
+
+This repository is deprecated; No further Issues or Pull Requests will be considered or approved.
+
+Further interest should be directed to the new repository that represents us a parachain. If you'd like to check out the latest version of the source code, report any issues, contribute any improvements, or download the latest release, please do so at:
```

Comment:
`represent us a parachain`: not quite clear what you mean here.

Reply:
Meant the new repo is more relevant to our current parachain build... I've removed it now.